sqlmesh/utils/metaprogramming.py

```diff
@@ -395,24 +413,31 @@ def is_value(self) -> bool:
         return self.kind == ExecutableKind.VALUE
 
     @classmethod
-    def value(cls, v: t.Any) -> Executable:
-        return Executable(payload=repr(v), kind=ExecutableKind.VALUE)
+    def value(cls, v: t.Any, is_metadata: t.Optional[bool] = None) -> Executable:
+        return Executable(payload=repr(v), kind=ExecutableKind.VALUE, is_metadata=is_metadata)
 
 
-def serialize_env(env: t.Dict[str, t.Any], path: Path) -> t.Dict[str, Executable]:
+def serialize_env(
+    env: t.Dict[str, t.Any],
+    env_metadata: t.Set[str],
+    path: Path,
+) -> t.Dict[str, Executable]:
     """Serializes a python function into a self contained dictionary.
 
     Recursively walks a function's globals to store all other references inside of env.
 
     Args:
         env: Dictionary to store the env.
+        env_metadata: Keys that correspond to "metadata only" objects in the env.
         path: The root path to seralize. Other modules will not be walked and treated as imports.
     """
     serialized = {}
 
     for k, v in env.items():
+        is_metadata = True if k in env_metadata else None
```

Comment:
None and not False?

Reply:
So you don't have redundant `False` values in the serialized `Executable`, `None` is dropped when doing `json()`.